### PR TITLE
DbUtil class implemented

### PR DESCRIPTION
**Pull request by srinivasreddygopu1812**
please review this

### CODE REVIEW

File: src/main/java/com/dbutil/DbUtil.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.dbutil;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class DbUtil {
+	
+	public static SessionFactory getSessionFactory() {
+		
+return		new Configuration().configure().buildSessionFactory();
```
Comment (rsree20) [inline src/main/java/com/dbutil/DbUtil.java]: reduce space here

Comment (rsree20) [inline src/main/java/com/dbutil/DbUtil.java]: reduce space here

File: src/main/java/com/dbutil/DbUtil.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.dbutil;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class DbUtil {
```
Comment (rsree20) [inline src/main/java/com/dbutil/DbUtil.java]: change class name

File: src/main/java/com/dbutil/DbUtil.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.dbutil;
+
```
Comment (rsree20) [inline src/main/java/com/dbutil/DbUtil.java]: space remove here too..

File: src/main/java/com/dbutil/DbUtil.java (hunk 2 above)
Comment (srinivasreddygopu1812) [reply src/main/java/com/dbutil/DbUtil.java]: ok i will change it

**Pull request merged**